src/server/generator.py

```diff
@@ -37,10 +40,14 @@ def generate(program, language, colour):
                skill_nums[1], language_skill,
                skill_nums[2], colour_skill)
 
+    print(final)
+    print(final)
+    print(final)
+    print(final)
```

Comment:
???

Reply:
Did I really add this?? I don't remember doing that at all. 🤔
Will remove in next PR (I'm having final be a dictionary instead that is returned)